### PR TITLE
fix a todo: rename parameters

### PR DESCRIPTION
**Pull request by zzl200012**
### What problem does this PR solve?

Issue Number: #325  

Problem Summary:

### What is changed and how it works?

What's Changed: parameters' name

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/mds/nameserver2/curvefs.cpp [hunk 1]
```diff
@@ -860,58 +860,56 @@ StatusCode CurveFS::CheckFileCanChange(const std::string &fileName,
     return StatusCode::kOK;
 }
 
-// TODO(hzchenwei3): change oldFileName to sourceFileName
-//                   and newFileName to destFileName)
-StatusCode CurveFS::RenameFile(const std::string & oldFileName,
-                               const std::string & newFileName,
+StatusCode CurveFS::RenameFile(const std::string & sourceFileName,
+                               const std::string & destFileName,
                                uint64_t oldFileId, uint64_t newFileId) {
```
Comment (cw123) [inline src/mds/nameserver2/curvefs.cpp:865]: oldFileId -> sourceFileId
newFileId -> destFileId

File: src/mds/nameserver2/curvefs.cpp [hunk 2]
```diff
@@ -860,58 +860,56 @@ StatusCode CurveFS::CheckFileCanChange(const std::string &fileName,
...
-    if (!oldFileName.compare(newFileName)) {
-        LOG(INFO) << "rename same name, oldFileName = " << oldFileName
-                  << ", newFileName = " << newFileName;
+    if (!sourceFileName.compare(destFileName)) {
+        LOG(INFO) << "rename same name, sourceFileName = " << sourceFileName
+                  << ", destFileName = " << destFileName;
         return StatusCode::kFileExists;
     }
 
     FileInfo  oldFileInfo;
```
Comment (cw123) [inline src/mds/nameserver2/curvefs.cpp:876]: the fileinfo is also need rename

File: src/mds/nameserver2/curvefs.cpp [hunk 3]
```diff
@@ -956,7 +954,7 @@ StatusCode CurveFS::RenameFile(const std::string & oldFileName,
         recycleFileInfo.set_parentid(RECYCLEBININODEID);
         recycleFileInfo.set_filename(recycleFileInfo.filename() + "-" +
                 std::to_string(recycleFileInfo.id()));
-        recycleFileInfo.set_originalfullpathname(newFileName);
+        recycleFileInfo.set_originalfullpathname(destFileName);
 
         // rename!
         FileInfo newFileInfo;
```
Comment (cw123) [inline src/mds/nameserver2/curvefs.cpp:960]: the fileinfo is also need rename

**Review by cw123 — CHANGES_REQUESTED** [review]

File: src/mds/nameserver2/curvefs.cpp (hunk 1 above)
Comment (zzl200012) [reply src/mds/nameserver2/curvefs.cpp]: fix

File: src/mds/nameserver2/curvefs.cpp (hunk 2 above)
Comment (zzl200012) [reply src/mds/nameserver2/curvefs.cpp]: fix

File: src/mds/nameserver2/curvefs.cpp (hunk 3 above)
Comment (zzl200012) [reply src/mds/nameserver2/curvefs.cpp]: fix

**Review by opencurveadmin — DISMISSED** [review]

**Review by cw123 — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**